roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -59,6 +59,16 @@
   failed_when: "'already exists' not in osmiq_perm_task.stderr and osmiq_perm_task.rc != 0"
   changed_when: osmiq_perm_task.rc == 0
 
+- name: Configure 3_2 role/user permissions
+  command: >
+    {{ openshift.common.admin_binary }} {{item}}
+    --config={{manage_iq_tmp_conf}}
+  with_items: "{{manage_iq_openshift_3_2_tasks}}"
+  register: osmiq_perm_3_2_task
+  failed_when: osmiq_perm_3_2_task.rc != 0
+  changed_when: osmiq_perm_3_2_task.rc == 0
```

Comment:
> Does this not only tell me that the role exists? I need to know if it is associated to the serviceAccounts management-infra/management-admin

Why don't you just unconditionally bind it?  The operation should be idempotent: `oadm policy add-cluster-role-to-user `.